controllers/onload_controller_test.go

```diff
@@ -791,36 +792,51 @@ var _ = Describe("onload controller", func() {
 			Entry("Upgrade Onload with SFC", &onloadv1alpha1.SFCSpec{}, &onloadv1alpha1.SFCSpec{}),
 		)
 
-		Describe("Testing Device Plugin options", func() {
-			It("shouldn't add anything when empty", func() {
-				devicePlugin := appsv1.DaemonSet{}
-				devicePluginName := types.NamespacedName{
-					Name:      onload.Name + "-onload-device-plugin-ds",
-					Namespace: onload.Namespace,
+		checkDevicePluginOptions := func(spec *onloadv1alpha1.DevicePluginSpec, devicePlugin *appsv1.DaemonSet) {
```

Comment:
Shall we remove `checkDevicePluginOptions()`, e.g. https://github.com/Xilinx-CNS/kubernetes-onload/commit/bed2e0c9a4baf2ca45b0db5b7c2f1b0db5ce4672?